dashboards/gp2gp_in_progress_sla_trending_dashbaord.json

```diff
@@ -0,0 +1,397 @@
+    "tokens": {
+      "default": {
+        "time_period": {
+          "value": "None"
+        }
+      }
+    }
+  },
+  "inputs": {
+    "input_x1LK7THX": {
+      "options": {
+        "defaultValue": 0,
+        "token": "cutoff",
+        "min": 0,
+        "max": 14
+      },
+      "title": "Cutoff (days)",
+      "type": "input.number"
+    },
+    "input_udP1B2BG": {
+      "options": {
+        "defaultValue": "0,",
+        "token": "time_range"
+      },
+      "title": "Time Range",
+      "type": "input.timerange"
+    },
+    "input_wLqPblo4": {
+      "options": {
+        "items": [
+          {
+            "label": "Month",
+            "value": "month"
+          },
+          {
+            "label": "Week",
+            "value": "week"
+          },
+          {
+            "label": "Day",
+            "value": "day"
+          }
+        ],
+        "token": "time_period",
+        "defaultValue": "month"
+      },
+      "title": "Time period",
+      "type": "input.dropdown"
+    }
+  },
+  "layout": {
+    "type": "absolute",
+    "options": {
+      "display": "auto",
+      "backgroundColor": "#ffffff",
+      "height": 2200,
+      "showTitleAndDescription": false
+    },
+    "structure": [
+      {
+        "item": "viz_Yrz79jnn",
+        "type": "block",
+        "position": {
+          "x": 30,
+          "y": 410,
+          "w": 1130,
+          "h": 380
+        }
+      },
+      {
+        "item": "viz_RYPhSk8c",
+        "type": "block",
+        "position": {
+          "x": 10,
+          "y": 10,
+          "w": 920,
+          "h": 100
+        }
+      },
+      {
+        "item": "viz_nrRMTa9E",
+        "type": "block",
+        "position": {
+          "x": 30,
+          "y": 1020,
+          "w": 1130,
+          "h": 380
+        }
+      },
+      {
+        "item": "viz_ysvkHFbs",
+        "type": "block",
+        "position": {
+          "x": 30,
+          "y": 220,
+          "w": 1130,
+          "h": 190
+        }
+      },
+      {
+        "item": "viz_VXusQloW",
+        "type": "block",
+        "position": {
+          "x": 30,
+          "y": 810,
+          "w": 1130,
+          "h": 210
+        }
+      },
+      {
+        "item": "viz_KKkIX6xn",
+        "type": "block",
+        "position": {
+          "x": 270,
+          "y": 110,
+          "w": 260,
+          "h": 90
+        }
+      },
+      {
+        "item": "viz_C7qrM833",
+        "type": "block",
+        "position": {
+          "x": 40,
+          "y": 1430,
+          "w": 1120,
+          "h": 610
+        }
+      },
+      {
+        "item": "viz_iodmYizw",
+        "type": "block",
+        "position": {
+          "x": 10,
+          "y": 110,
+          "w": 260,
+          "h": 90
+        }
+      }
+    ],
+    "globalInputs": [
+      "input_udP1B2BG",
+      "input_x1LK7THX",
+      "input_wLqPblo4"
+    ]
+  },
+  "description": "",
+  "title": "gp2gp_in_progress_sla_trending_dashboard"
+}
```

Comment:
Should we add a link to the Snapshot dashboard?

Reply:
added